packages/project-access/package.json

```diff
@@ -36,11 +36,13 @@
         "@sap-ux/ui5-config": "workspace:*",
         "fast-xml-parser": "4.2.7",
         "findit2": "2.2.3",
+        "json-parse-even-better-errors": "3.0.2",
         "mem-fs": "2.1.0",
         "mem-fs-editor": "9.4.0",
         "semver": "7.5.4"
     },
     "devDependencies": {
+        "@types/json-parse-better-errors": "1.0.3",
```

Comment:
Sorry @broksy, seems this does not contain correct types, I've misread the package name, we are using json-parse-**even**-better-errors, so we can remove this and keep your declaration file.
```suggestion
```